README.md

```diff
@@ -70,7 +70,10 @@ The tool will leverage REST client API to request to Atlas which requires authen
 
 Given the approach is insecure regardless of security mode of cluster, we strongly encourage you to pass `--interactive-auth` as parameter, which asks you to input username and password of Atlas interactively.
 
-After running above command, you can set `atlas.client.checkModelInStart=false` in `atlas-application.properties` to skip checking and creating models in Spark Atlas Connector's startup.
+If you would like to let SAC handles it instead, you can set you can set `atlas.client.checkModelInStart=true` in `atlas-application.properties` to let SAC check and create models in Spark Atlas Connector's each startup.
+Please note that it leverages REST API even you configure `atlas.client.type=kafka`, and it would require additional configuration for authentication. (`atlas.client.username` and `atlas.client.password`)
+
+In the meanwhile, we're also working on migrating models into Apache Atlas: once it's done, we no longer require this step and leverage the models Atlas registers for the startup.
```

Comment:
Does it mean any changes to the model has to go via Atlas team ? Could we just keep the way for Spark to load the models by running the command?

Reply:
According to the Atlas team, that's the standard way for dealing with models registration, and decision has been made. (It would be also easier for us to ensure the necessary models are available in Atlas.)

We might want to keep that feature for end users who wants to use the Atlas version which doesn't ship the Spark models, but maybe it's better to not keep two different approaches together which could be out of sync.